src/services/user.ts

```diff
@@ -0,0 +1,5 @@
+import { User } from '../interfaces/User';
+
+export const getUserById = (users: User[], id: number): User => {
+  return users.find(user => user.id === id) as User;
```

Comment:
```suggestion
  return users.find(user => user.id === id) || null;
```
try to avoid assertions as much as you can